src/codeflare_sdk/cluster/cluster.py

```diff
@@ -152,6 +152,7 @@ def create_app_wrapper(self):
         write_to_file = self.config.write_to_file
         local_queue = self.config.local_queue
         labels = self.config.labels
+        kueue = self.config.kueue
```

Comment:
Is the `kueue` parameter really necessary? I fail to see why `local_queue` is not enough. And if there is a "default" LocalQueue, then we can assume Kueue should be used and the label set?

Reply:
Perhaps I was too gun ho with this...
I will adjust the PR to match what you said above good catch Antonin